src/foam/u2/detail/SectionView.js

```diff
@@ -78,6 +78,14 @@ foam.CLASS({
           return this.getNestedPropValue(obj[props.shift()], props.join('.'))
         }
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'loadLatch',
+      preSet: function(o, n) {
+        return ! ( o || n );
```

Comment:
thanks for a suggestion @kgrgreer 
looks  really pretty now

Reply:
Is that correct? Should it be just 'o || n'? I see. You start off with value: true. Normally latches go from false to true, so can you reverse the logic?

Reply:
okay, so setting it to true was a bug. as the default value of loadLatch would make section which are used elsewhere not only in TabbedDetailView blank. so we don't need to reverse